doc/source/dev/contributor/adding_notebooks.rst

```diff
@@ -0,0 +1,146 @@
+.. _adding-notebooks:
+
+Adding or editing tutorials as Jupyter notebooks
+------------------------------------------------
+
+Under the ``doc/source/`` folder of the SciPy tree you can find a few
+documents written in MyST markdown format. The content in these files is
+executed when the SciPy documentation is built (locally or on CI) and any
+outputs generated by the execution are rendered in the final HTML files. In
+addition, they can be made interactive in the SciPy documentation website by
+using the
+`jupyterlite-sphinx <https://jupyterlite-sphinx.readthedocs.io/en/latest/>`__
+extension.
+
+If you have a document written in Jupyter notebook format (an ``.ipynb`` file)
+and would like to submit it as part of the SciPy documentation, there are two
+options: you can convert it into a MyST Markdown file, and work with a ``.md``
+file only, or you can pair your ``.ipynb`` file with a ``.md`` file and work
```

Comment:
It might be worth mentioning:

```
In this document, we address conversion to a MyST Markdown file. For more information about MyST-NB, Jupytext and pairing notebooks, you can also consult the `Pairing tutorial on NumPy Tutorials <https://numpy.org/numpy-tutorials/content/pairing.html>`__. For even more information, please consult the
`MyST-NB documentation <https://myst-nb.readthedocs.io/en/latest/authoring/text-notebooks.html>`__.
```